core/src/main/scala/pureconfig/error/ConfigReaderFailures.scala

```diff
@@ -52,6 +52,6 @@ case class ConfigReaderFailures(head: ConfigReaderFailure, tail: List[ConfigRead
 
 object ConfigReaderFailures {
 
-  def apply(configReaderFailure: ConfigReaderFailure): ConfigReaderFailures =
-    new ConfigReaderFailures(configReaderFailure, List.empty[ConfigReaderFailure])
+  def apply(first: ConfigReaderFailure, rest: ConfigReaderFailure*): ConfigReaderFailures =
```

Comment:
I'm afraid the type signature of this method will be almost the same as the case class constructor (the compiler is only ok with it because this is a `Seq` after erasure, while the case class constructor is a `List`). Do you think this might create some issues in the future? The alternative is to name this anything other than `apply`.

If you think this is ok, LGTM 👍

Reply:
Good point. What do you think of having a single `apply` (see 8975dd1)? I believe part of the motivation for having both was to have more convenient pattern matches on `ConfigReaderFailures`, but I suspect that's not a frequent use case outside of tests, and the available `toList` conversion can also help with that.